include/lbann/execution_algorithms/kfac/kfac_block.hpp

```diff
@@ -204,6 +264,12 @@ class kfac_block {
       TODO: Remove this. */
   const size_t m_layer_id;
 
+  /** @brief Enable copying of errors to enhance async communication. */
+  bool m_enable_copy_errors;
+
+  /** @brief Enable copying of activations to enhance async communication. */
+  bool m_enable_copy_activations;
+
```

Comment:
We don't really optimize our packing, but putting two `bool`s between a `size_t` and an `int` effectively guarantees 2 extra pad bytes/object... I would move these with `m_has_kronecker_inverse` below, or just move all 3 `bool`s to the end of the struct.